lib/matplotlib/backends/backend_pgf.py

```diff
@@ -639,7 +639,8 @@ def draw_image(self, gc, x, y, im, transform=None):
         fname = os.path.splitext(os.path.basename(self.fh.name))[0]
         fname_img = "%s-img%d.png" % (fname, self.image_counter)
         self.image_counter += 1
-        _png.write_png(im[::-1], os.path.join(path, fname_img))
+        with pathlib.Path(path, fname_img).open("wb") as file:
```

Comment:
I don't quite like assigning to builtin names, but acknowledge that there's already precedence for this in the library.

Reply:
It's not a builtin name anymore...